src/cow-react/pages/Account/LockedGnoVesting/claimData/index.ts

```diff
@@ -13,14 +12,14 @@ interface Claim {
 
 const indexFiles = {
   [SupportedChainId.MAINNET]: mainnetIndex,
-  [SupportedChainId.RINKEBY]: rinkebyIndex,
+  // [SupportedChainId.RINKEBY]: rinkebyIndex,
```

Comment:
You can remove this completely

Reply:
Thanks! Deleted and kept commented only for MOD files